default.nix

```diff
@@ -0,0 +1,10 @@
+(import
+  (
+    let lock = builtins.fromJSON (builtins.readFile ./flake.lock); in
+    fetchTarball {
+      url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+      sha256 = lock.nodes.flake-compat.locked.narHash;
+    }
+  )
+  { src = ./.; }
+).defaultNix
```

Comment:
It's probably best to just have either `default.nix` or `shell.nix`, and not both? `nix-shell` supports both, but prefers `shell.nix` - https://nixos.org/manual/nix/stable/command-ref/nix-shell.html

Reply:
Yes, we can live without a `shell.nix`. Without `default.nix` packages attributes can't be built.